core/hub/settings.go

```diff
@@ -0,0 +1,52 @@
+package hub
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/TRON-US/go-btfs/repo"
+	hubpb "github.com/tron-us/go-btfs-common/protos/hub"
+	nodepb "github.com/tron-us/go-btfs-common/protos/node"
+	"github.com/tron-us/go-btfs-common/utils/grpc"
+
+	"github.com/ipfs/go-datastore"
+)
+
+var (
+	keyFormat = "/btfs/%s/settings/v1"
```

Comment:
Why a new format? It should share the same `GetHostStorageKey` function.

If you want a new format, you need to update the other calls as well.

Reply:
i try to refactor the settings and hosts storage.
and i think maybe it's better to seperate setting and hosts storage.
settings.go somehow like a setting dao, and i'll add a hosts.go later.

Reply:
for settings, i think i have update all the calls.

Reply:
Oh that's fine to have a new format, but I'd prefer having a function to do it rather than `keyFormat` and sprintf. That's fine, maybe when you add hosts.go later can include that.